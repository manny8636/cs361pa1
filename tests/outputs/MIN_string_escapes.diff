--- outputs/MIN_string_escapes.txt	2022-02-13 13:37:46.683549286 -0500
+++ expected/MIN_string_escapes.txt	2021-01-25 15:27:24.000000000 -0500
@@ -1 +1,2 @@
+VALUE [string]: 'This "is" okay to do. Here is a \ backslash.'
 Success!
