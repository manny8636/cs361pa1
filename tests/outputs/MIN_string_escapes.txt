Parsing ../data/string_escapes.txt failed
