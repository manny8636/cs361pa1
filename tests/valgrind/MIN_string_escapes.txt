==146753== Memcheck, a memory error detector
==146753== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==146753== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==146753== Command: ../json -v ../data/string_escapes.txt
==146753== 
Parsing ../data/string_escapes.txt failed
==146753== 
==146753== HEAP SUMMARY:
==146753==     in use at exit: 50 bytes in 1 blocks
==146753==   total heap usage: 4 allocs, 3 frees, 16,906 bytes allocated
==146753== 
==146753== LEAK SUMMARY:
==146753==    definitely lost: 50 bytes in 1 blocks
==146753==    indirectly lost: 0 bytes in 0 blocks
==146753==      possibly lost: 0 bytes in 0 blocks
==146753==    still reachable: 0 bytes in 0 blocks
==146753==         suppressed: 0 bytes in 0 blocks
==146753== Rerun with --leak-check=full to see details of leaked memory
==146753== 
==146753== For lists of detected and suppressed errors, rerun with: -s
==146753== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
