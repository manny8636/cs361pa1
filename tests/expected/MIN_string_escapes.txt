VALUE [string]: 'This "is" okay to do. Here is a \ backslash.'
Success!
